server/routes/findInterventions/findInterventionsController.ts

```diff
@@ -20,7 +21,7 @@ export default class FindInterventionsController {
     const presenter = new SearchResultsPresenter(interventions, filter, pccRegions)
     const view = new SearchResultsView(presenter)
 
-    res.render(...view.renderArgs)
+    ControllerUtils.renderWithLayout(res, view)
```

Comment:
So just to clarify, we'll be using this everywhere we render a view (rather than `res.render` as that's wrapped in this function?)

Reply:
Yep, that’s right - at least, if you want to render a view that uses our `layout.njk` layout.